docs/articles_en/openvino-workflow/torch-compile.rst

```diff
@@ -20,6 +20,186 @@ By default, Torch code runs in eager-mode, but with the use of ``torch.compile``
 How to Use
 ####################
 
+
+.. tab-set::
+
+   .. tab-item:: Image Generation
+
+      .. tab-set::
+
+         .. tab-item:: Stable-Diffusion-2
+
+            .. code-block:: py
+               :force:
+
+               import torch
+               from diffusers import StableDiffusionPipeline, DPMSolverMultistepScheduler
+
+               model_id = "stabilityai/stable-diffusion-2-1"
+
+               # Use the DPMSolverMultistepScheduler (DPM-Solver++) scheduler here instead
+               pipe = StableDiffusionPipeline.from_pretrained(model_id, torch_dtype=torch.float16)
+               pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config)
+
+               + pipe.text_encoder = torch.compile(pipe.text_encoder, backend="openvino") #Optional
+               + pipe.unet = torch.compile(pipe.unet, backend=“openvino”)
+               + pipe.vae.decode = torch.compile(pipe.vae.decode, backend=“openvino”) #Optional
+
+               prompt = "a photo of an astronaut riding a horse on mars"
+               image = pipe(prompt).images[0]
+
+               image.save("astronaut_rides_horse.png")
+
+
+         .. tab-item:: Stable-Diffusion-3
+
+            .. code-block:: sh
+
+               import torch
+               from diffusers import StableDiffusion3Pipeline
+
+               pipe = StableDiffusion3Pipeline.from_pretrained("stabilityai/stable-diffusion-3-medium-diffusers", torch_dtype=torch.float32)
+
+               + pipe.transformer = torch.compile(pipe.transformer, backend="openvino")
+
+               image = pipe(
+                   "A cat holding a sign that says hello world",
+                   negative_prompt="",
+                   num_inference_steps=28,
+                   guidance_scale=7.0,
+               ).images[0]
+
+               image.save('out.png')
+
+         .. tab-item:: Stable-Diffusion-XL
+
+            .. code-block:: sh
+
+               import torch
+               from diffusers import UNet2DConditionModel, DiffusionPipeline, LCMScheduler
+
+               unet = UNet2DConditionModel.from_pretrained("latent-consistency/lcm-sdxl", torch_dtype=torch.float16, variant="fp16")
+               pipe = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", unet=unet, torch_dtype=torch.float16, variant="fp16")
+               pipe.scheduler = LCMScheduler.from_config(pipe.scheduler.config)
+
+               + pipe.text_encoder = torch.compile(pipe.text_encoder, backend="openvino") #Optional
+               + pipe.unet = torch.compile(pipe.unet, backend="openvino")
+               + pipe.vae.decode = torch.compile(pipe.vae.decode, backend="openvino") #Optional
+
+               prompt = "a close-up picture of an old man standing in the rain"
+               image = pipe(prompt, num_inference_steps=5, guidance_scale=8.0).images[0]
+               image.save("result.png")
+
+   .. tab-item:: Image Generation
```

Comment:
These are text generation models. can you please change the tab to Text generation?